sr_logging_common/logging/bag_rotate.py

```diff
@@ -40,29 +27,20 @@ def __init__(self, path, desired_bag_number, prefix):
         self.run()
 
     def remove_old_bags(self):
-        bags_to_remove = self.get_suffixed_bags(".bag")
-        bags_to_remove.extend(self.get_suffixed_bags(".bag.orig.active"))
+        bags_to_remove = self.get_file_paths_by_key(".bag")
         bags_to_remove.sort()
 
         for i, bag in enumerate(bags_to_remove):
             if i < len(bags_to_remove)-self._desired_bag_number - 1:
                 remove(bag)
 
-    def get_suffixed_bags(self, suffix):
+    def get_file_paths_by_key(self, key):
         all_files = listdir(self._path)
-        all_bags = [s_file for s_file in all_files if s_file.endswith(suffix)]
-        suffixed_bags = []
-        for bag in all_bags:
-            actual_prefix_parts = set([t for t in bag.split("_") if t.isalpha()])
-            if self._desired_prefix_parts == actual_prefix_parts:
-                suffixed_bags.append(f"{self._path}/{bag}")
-        return suffixed_bags
+        matching_files = [f"{self._path}/{s_file}" for s_file in all_files if key in s_file]
```

Comment:
s_file isn't a meaningful name and using os.join is preferred to manual concatenation. I remember there was an issue with doing this rotation when there are still bag.orig.active files

Reply:
good to hear from you! I changed the names and added the join, very good idea!